### ISSUE TITLE
About Localization Tasks

### ISSUE THREAD

**Issue by huifenghc** [wolo-wolo/FSFM#4]
Hi, thank you for your excellent work! I really appreciate the clarity of your codebase and the impressive performance demonstrated in your paper.
﻿
I have a question regarding the learned representations. Do you think the features extracted by your model could be effectively adapted for downstream tasks such as face forgery localization? For instance, instead of only classifying an image as real or fake, could the learned representations be used to localize forged regions in manipulated facial images?
﻿
Any insights or suggestions on how to adapt the current framework for this purpose would be greatly appreciated!
﻿
Thanks again for the great work.

**Comment by wolo-wolo** [comment]
Thank you for being interested in our work.  
A good issue. 
Although we explored the detection (classification ) capabilities of various face security tasks, unfortunately, we did not conduct location experiments due to time constraints.
1) I think FSFM has some potential for face forgery localization, given that it explicitly utilizes pixel-level facial region parsing during pre-training.
2) Just as fine-tuning in downstream classification tasks requires data and category labels, to achieve forged region localization in face images, you can use the pre-trained FSFM weights to initialize the ViT network as the backbone for your location model, and train it on the localization data with labels.

**Comment by huifenghc** [comment]
Well, thanks for your help.

**Comment by wolo-wolo** [comment]
Dear @huifenghc 
The extended version of our work is now released.
You can find everything below：
[project page](https://fsfm-3c.github.io/fsvfm.html)
[paper of extended version](https://arxiv.org/pdf/2510.10663)
[full codes and models](https://github.com/wolo-wolo/FSFM-CVPR25/tree/FSVFM-extension)

Thank you so much for your interest — we hope these will be useful for your research or applications. We look forward to seeing what you build with them!

Best regards
